charts/pokt-portal/templates/hpa.yaml

```diff
@@ -10,6 +10,17 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: {{ include "pokt-portal.fullname" . }}
+  behavior:
```

Comment:
we should include a conditional here , so if autoscaling = true (as with our prod release), it will only add the behaviour block if `.Values.autoscaling.behavior` is provided. other wise it should be ommitted from the HPA resource. This chart would break prod when it gets release because it will expect certain values because `.Values.autoscaling.enabled = true`

Reply:
Roger that.

Reply:
Comments addressed.

Reply:
Comments addressed.